operations/iuf/workflows/upgrade.md

```diff
@@ -0,0 +1,88 @@
+# Upgrade
+
+The following workflow describes an upgrade procedure for CSM as well as all non-CSM product
+content provided via a HPC CSM Software Recipe release. The steps documented here alternate
+between CSM upgrade instructions unrelated to IUF and non-CSM product upgrade instructions managed
+by IUF.
+
+All stages of `iuf` are executed in this workflow: all of the new product software provided in the
+recipe release is deployed and all management NCNs and managed compute and application nodes are
+rebooted to new images and CFS configurations. Manual operations are documented for procedures that
+are not currently managed by IUF.
+
+The upgrade workflow comprises the following procedures which must be executed in the order shown.
+
+1. CSM preparation, Stage 0.1, and Stage 0.2
+
+   Read the _Important Notes_ section of the
+   [CSM 1.3.0 or later to 1.4.0 Upgrade Process](../../../upgrade/Upgrade_Management_Nodes_and_CSM_Services.md)
+   documentation and then follow these CSM instructions in order:
+
+   1. [Prepare for Upgrade](../../../upgrade/prepare_for_upgrade.md)
+   1. [Stage 0.1 - Prepare assets](../../../upgrade/Stage_0_Prerequisites.md#stage-01---prepare-assets)
+   1. [Stage 0.2 - Prerequisites](../../../upgrade/Stage_0_Prerequisites.md#stage-02---prerequisites)
+
+1. Download product media and prepare for IUF upgrade
+
+   Follow the IUF [Prepare for the install or upgrade](preparation.md) instructions
+
+1. Product Delivery
+
+   Follow the IUF [Product delivery](product_delivery.md) instructions
+
+1. Configuration
+
+   Follow the IUF [Configuration](configuration.md) instructions
+
+1. Image preparation
+
+   Follow the IUF [Image preparation](image_preparation.md) instructions
+
+1. CSM Stage 0.4
+
+   Follow the CSM
+   [Stage 0.4 - Backup workload manager data](../../../upgrade/Stage_0_Prerequisites.md#stage-04---backup-workload-manager-data)
+   instructions
+
+1. Backup
+
+   Follow the IUF [Backup](backup.md) instructions
+
+1. Management Rollout
+
+   Follow the IUF [Management rollout](management_rollout.md) instructions
+
+   **`UNRESOLVED ISSUE`** Does IUF cover all of the following [Stage 1](../../../upgrade/Stage_1.md)
```

Comment:
@haasken-hpe @leliasen-hpe @dzou-hpe I will need your help to resolve this.

Reply:
no

Reply:
They way you have this now looks good to me, but I'm not that familiar with stage 1 of the CSM upgrade.

Reply:
The next commit links to this procedure inline with the existing instructions in the workflows/management_rollout.md file.